airflow/utils/pydantic.py

```diff
@@ -24,19 +24,15 @@
 
 from __future__ import annotations
 
+from importlib.metadata import PackageNotFoundError, version
```

Comment:
I'm assuming multiple packages provide a `version` function which potentially does different things. IMHO, it could improve readability if imported `as version_string` or `as get_version_string`.

Reply:
well version is a method in stdlib's importlb.metadata, I think it makes little sense to rename it - but I see your point. likely better to import `from importlib import metadata` and use the package explicitly. Let me change it.